src/ophyd_async/plan_stubs/_fly.py

```diff
@@ -129,17 +127,12 @@ def fly_and_collect(
     done = False
     while not done:
         try:
-            yield from bps.wait(group=group, timeout=0.5)
+            yield from bps.wait(group=group, timeout=0.5, move_on=True)
         except TimeoutError:
```

Comment:
I actually don't think we need to do this now. It seems timeout is only raised if `move_on=False` I think we can change this whole thing to:

``` python
while (
    yield from bps.wait(
        group=group,
        timeout=_SECONDS_TO_FORCE_COLLECT_AFTER,
        move_on=True
    )
):
    yield from bps.collect(*detectors, name=stream_name)
```

Since done is returned here:
https://github.com/bluesky/bluesky/blob/978995b69c00cc0b9bef5103bdb31b665a6fbed9/src/bluesky/run_engine.py#L2379